### ISSUE TITLE
woocommerce.scss box-sizing warning message

### ISSUE THREAD

**Issue by themestechnology** [woothemes/woocommerce#8752]
This message has been appearing for quite a while not when I build my SCSS files using grunt is there something you can do to stop this from happening as I have to use --force to build everything.

WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
         on line 410 of /home/xxxx/node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss, in `box-sizing'
         from line 892 of bower_components/woocommerce/assets/css/woocommerce.scss
         from line 4 of scss/foundation.scss

WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
         on line 410 of /home/xxxx/node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss, in `box-sizing'
         from line 1280 of bower_components/woocommerce/assets/css/woocommerce.scss
         from line 4 of scss/foundation.scss

WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
         on line 410 of /home/xxxx/node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss, in `box-sizing'
         from line 248 of bower_components/woocommerce/assets/css/prettyPhoto.scss
         from line 5 of scss/foundation.scss

WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
         on line 410 of /home/xxxx/node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss, in `box-sizing'
         from line 312 of bower_components/woocommerce/assets/css/woocommerce-layout.scss
         from line 6 of scss/foundation.scss

**Comment by jameskoster** [comment]
Looks like this is coming from your `scss/foundation.scss` file which is not a WooCommerce core file.

Also, have you done an `npm update` recently to make sure you have the latest version of bourbon?

**Comment by themestechnology** [comment]
No "from line 4 of scss/foundation.scss" was the line where the file was @imported from.

I have moved the woo files directly to by Gruntfile.js and am still getting exactly the same error without the foundation.scss reference.

I have also updated everything, and removed the npm and bower folders so all files are fully up do date.

>apt-get update;
>npm update;
>bundle update;
>bower update;

WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
         on line 410 of /home/xxxx/node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss, in `box-sizing'
         from line 892 of bower_components/woocommerce/assets/css/woocommerce.scss
